Provider/MultiUploadInterface.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Provider;
+
+use Sonata\AdminBundle\Form\FormMapper;
+
+/**
+ * @author Maximilian Behrsing <m.behrsing@gmail.com>
+ */
+interface MultiUploadInterface
+{
+    /*
+     * @param FormMapper $formMapper
+     *
+     * @return mixed
+     */
+    public function configureMultiUpload(FormMapper $formMapper);
+
+    /*
+     * @return bool
+     */
+    public function isMultiUploadEnabled();
```

Comment:
> Could there be a case where I want to enable it for images, but not for PDF's for example?

Looks like feature creep. It can be added afterwards if necessary.